setting/src/main/res/layout/fragment_about.xml

```diff
@@ -1,45 +1,50 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical"
-    tools:context="jp.bellware.echo.view.setting.AboutFragment">
+    android:layout_height="match_parent">
 
-    <ImageView
+    <LinearLayout
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/tfandkusu/quickecho/blob/9c847e5c9869beb9b9857cca0d9fc5b6229fb843/setting/src/main/res/layout/fragment_about.xml#L6"></span>This <code>LinearLayout</code> layout or its <code>FrameLayout</code> parent is useless</del></td>
    </tr>
  </tbody>
</table>